pgcli/packages/pgspecial.py

```diff
@@ -1025,6 +1026,44 @@ def toggle_timing(cur, arg, verbose):
     message += "on." if TIMING_ENABLED else "off."
     return [(None, None, None, message)]
 
+def list_named_queries(cur, arg, verbose):
+    """Returns (title, rows, headers, status)"""
+    if not verbose:
+        rows = [[r] for r in namedqueries.list()]
+        headers = ["Name"]
+    else:
+        headers = ["Name", "Query"]
+        rows = [[r, namedqueries.get(r)] for r in namedqueries.list()]
+    return [('', rows, headers, "")]
+
+def save_named_query(cur, arg, verbose):
+    """Returns (title, rows, headers, status)"""
+    name, query = arg.split(' ', 1)
+    namedqueries.save(name, query)
+    return [(None, None, None, "Saved.")]
+
+def delete_named_query(cur, arg, verbose):
+    namedqueries.delete(arg)
+    return [(None, None, None, "Deleted.")]
+
+def execute_named_query(cur, arg, verbose):
+    """Returns (title, rows, headers, status)"""
+    if arg == '':
+        return list_named_queries(cur, arg, verbose)
+
+    query = namedqueries.get(arg)
+    title = '> {}'.format(query)
```

Comment:
This type of string interpolation will fail in Python 2.6. Instead use

```
title = '> %s' % query
```

pgcli supports Python 2.6, 2.7, 3.3 and 3.4.